chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/ChatManagerWithFailingTokenProviderSpec.kt

```diff
@@ -21,7 +21,6 @@ import java.util.concurrent.Future
 import java.util.concurrent.atomic.AtomicInteger
 
 object ChatManagerWithFailingTokenProviderSpec : Spek({
-    afterEachTest(::closeChatManagers)
```

Comment:
For my own knowledge, would be interested to know why we've removed this line? Is it because `tearDownInstance` does the same thing as `closeChatManagers`?

Reply:
The tests here create ChatManagers only internally and close them after each test (L46). They don't use that facility of creating and tracking multiple chat managers within execution of a single test (for different users connecting) like end to end tests do. End to end tests do tracking of created chat managers so that they can close them after test.

This change has been inspired by the question you asked on the previous PR and I mentioned the intend to bring this cleanup in my next PR [there](https://github.com/pusher/chatkit-android/pull/152#issuecomment-578844322).